data/wanderer/wanderer missions.txt

```diff
@@ -0,0 +1,177 @@
+	on offer
+		conversation
+			`The two Wanderers from earlier are waiting for you in the spaceport, but this time Iriket, the terraforming director, is with them. The other two leave as she starts explaining the situation.`
+			`	"We have [restored, healed] the plant, and it is surviving here for now. But we will soon be moving out of these systems. This plant [originates, belongs] in Korath space, so we must find a way to return it there."`
+			`	Iriket pauses and thinks for a moment before continuing. "The Korath do not seem to me to be the type to [create, engineer] such a flower through [artificial, unnatural] means. I suspect it originates on one of their worlds. If we are fortunate, it may [grow, prosper] there still. From an initial look, we [believe, suspect] that it is native to either <planet>, Sopi Lefarkata, or possibly one of those worlds that is so [damaged, destroyed] that it is difficult to [see, imagine] what it might once have been like."`
+			choice
+				`	"You are really set on giving these worlds to the Unfettered, then?"`
+				`	"What do you suggest we do?"`
+					goto end
+			`	Iriket responds sharply. "That is our [purpose, way]. We heal worlds, and then we turn them over to other species. We would not [imagine, consider] any other course of action."`
+			label end
+			`	"Let us [search, visit] <planet> first. It is near to the worlds of the Mereti and not in the [territory, region] of their [rivals, foes], so it seems more likely that this plant [originated, grew] there."`
+				accept
+	on complete
+		dialog `Iriket guides you to an area of high, cool steppes as you approach. You find somewhere to land the ship, and she steps out. The air that blows in through the open door stings your eyes, but Iriket seems too deep in thought to notice. "It would have [grown, originated] here, I am certain of it. But it will do so no longer, not for many [years, cycles]." Iriket bows her head and holds one wing in front of herself silently for a moment before continuing. "We must find some world with similar [regions, environments] that we can introduce it to. Perhaps we will have better luck on Sopi Lefarkata."`
+
+mission "Wanderers: Mereti: The Plant 3"
+	landing
+	name "A home for the plant"
+	passengers 1
+	description "Bring Tema'a Iriket to the Korath planets she marked, to determine whether any of them is a suitable home for the plant from the Mereti."
+	source "Solima Skarati"
+	destination "Sopi Lefarkata"
+	to offer
+		has "Wanderers: Mereti: The Plant 2: done"
+	on complete
+		conversation
+			`This time, Iriket guides you down into an equatorial lowland - a dry grassy plain over which a cool wind blows. You can feel the similarity to the steppe you visited on Solima Skarati, but this time you can breathe freely without the air stinging your eyes and mouth. Iriket takes some soil and places it in a small device, looking at it for a moment before turning to you. "Here. It will [grow, prosper] here."`
```

Comment:
```suggestion
			`This time, Iriket guides you down into an equatorial lowland - a dry grassy plain over which a cool wind blows. You can feel its similarity to the steppe you visited on Solima Skarati, but this time you can breathe freely without the air stinging your eyes and mouth. Iriket takes some soil and places it in a small device, looking at it for a moment before turning to you. "Here. It will [grow, prosper] here."`
```